apps/zbugs/src/root.tsx

```diff
@@ -67,6 +70,16 @@ export function Root() {
 
   useSoftNav();
 
+  const login = useLogin();
+  const connectionState = useZeroConnectionState();
+
+  // if we're in needs-auth state, log out the user
+  useEffect(() => {
+    if (connectionState.name === ConnectionStatus.NeedsAuth) {
```

Comment:
I find it strange that we use connection*State* but the enum is called Connection*Status*.

I think we should rename the enum.